slsim/lens.py

```diff
@@ -16,6 +17,22 @@
 
 from slsim.lensed_system_base import LensedSystemBase
 
+# Set a global flag to track microlensing availability
+MICROLENSING_AVAILABLE = False
+
+try:
+    # This import is placed here bcoz otherwise it might cause issues with someone
+    # not having Luke's microlensing package installed!
+    from slsim.Microlensing.lightcurve import MicrolensingLightCurveFromLensModel
+
+    MICROLENSING_AVAILABLE = True
```

Comment:
it seems none of the tests with microlensing will be executed in the CI. Can you add the microlensing code in the test_requirements.txt?